packages/clock/main_test.go

```diff
@@ -0,0 +1,16 @@
+package clock
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/iotaledger/goshimmer/packages/testhelper"
```

Comment:
It's a good practice to avoid importing with ".":
```suggestion
	"github.com/iotaledger/goshimmer/packages/testhelper"
```

Reply:
question before I change:

We should never use `.`?
Can we automate this rule by the linter?

Reply:
Should we add this also to https://goshimmer.docs.iota.org/teamresources/guidelines.html?
Can you give examples of exceptions? (I just thought that if there should be exceptions to the rule, they should be in`test` packages)

Reply:
A few exceptions that I can come up with:
1. Indeed in test files, but in a different situation: when you have package "foo" and your unit tests are in the same directory but in package "foo_test" (good practice in Go), then you might want to import "foo" with "." to not add the prefix `foo.` to every function call.
2. When you have an internal package

But again, those exceptions are case by case and aren't critical, I would bother adding them as a rule to the guidelines.